Maintenance/deb/sid/debian/README.source

```diff
@@ -1,11 +0,0 @@
-normalize-audio for Debian
---------------------------
-
-This package uses quilt to manage all modifications to the upstream source.
-Changes are stored in the source package as diffs in debian/patches and
-applied during the build.
-
-See /usr/share/doc/quilt/README.source for a detailed explanation.
-
- -- Joachim Reichel <joachim.reichel@gmx.de>  Sun, 15 Jan 2012 14:38:04 +0100
-
```

Comment:
Why do you remove those files?! They are used by Joachim Reichel to maintain the Debian packages.

Reply:
It is automatic. Every time I create a new branch from master it does that.

Reply:
It seems to be a bug in a personnal convenience script. I'll look into it and fix it.